src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -22,4 +24,30 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public int hashCode() {
+        int result = 5;
+        result = 17 * result + (width == null ? 0 : width.hashCode())
```

Comment:
maek the same operation for each field: 
` result = 17 * result + (width == null ? 0 : width.hashCode());
 result = 17 * result + (length == null ? 0 : length.hashCode());
//so on
return result;`

Reply:
each time using the same variable result